compress_string.cpp

```diff
@@ -1,41 +1,34 @@
 #include <iostream>
-#include <vector>
-#include <string>
 #include <sstream>
+#include <string>
+#include <vector>
 
 /*
   Given a pointer to a string to compress
   return a string in the form:
   <char><char_count> of all the characters in the to_compress string
   eg. aaabbcccdff -> a3b2c3d1f2
 
+
   clang++-3.7 -std=c++11 compress_string.cpp -o compress_string
   ./compress_string <string_to_compress>
 */
 
-using namespace std;
-
-string compress(const string &to_compress)
+std::string compress(const std::string &to_compress)
```

Comment:
I would not use using namespace X as a matter of course.
It's OK in source files but never in headers => it's nice to consistently not use it.
Other people hold different opinions.